### DISCUSSION TITLE
any way to trigger via an API from my server?

### DISCUSSION THREAD

**Discussion by seanvosler** [jzillmann/pdf-to-markdown#46] [Q&A]
super noob here, not even sure if I formed the question correctly hah..
Basically I want to be able to send a PDF via API to pdf-to-markdown that's hosted on my server and return the markdown file results...

Any ideas?

**Comment by darkcheftar** [comment]
As Far as I know it is work in progress.

Hence the UI is being seperated from functionality

To make an npm package of the pdf2markdown,

Once its done I think you can achieve what you asking.

Am I right @jzillmann ?

**Reply by jzillmann** [reply]
Yes, but progress on the new version has stalled...
